lib/SILOptimizer/Mandatory/MandatoryInlining.cpp

```diff
@@ -784,7 +784,8 @@ runOnFunctionRecursively(SILOptFunctionBuilder &FuncBuilder,
                          DenseFunctionSet &FullyInlinedSet,
                          ImmutableFunctionSet::Factory &SetFactory,
                          ImmutableFunctionSet CurrentInliningSet,
-                         ClassHierarchyAnalysis *CHA) {
+                         ClassHierarchyAnalysis *CHA,
+                         SmallPtrSetImpl<SILFunction *> &changedFunctions) {
```

Comment:
I wouldn’t use a PtrSet for this. My suggestion instead would be use a small vector and in the caller call sortUnique. Iterating over Hash table backed sets is a code smell.

Reply:
I think the performance is more relevant here but, I don't feel strongly and the performance difference is probably minimal. I'll update it.